collection-scripts/gather_static_pod_monitor_logs

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Downloads the startup.log (and its rotated copies) from /var/logs/kube-apiserver on each master node
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+BASE_COLLECTION_PATH="${BASE_COLLECTION_PATH:-/must-gather}"
+
+echo "WARNING: Collecting one or more startup monitor logs on ALL masters in your cluster. This could take a large amount of time." >&2
+
+# the command executed by xargs below expects four parameters:
+# $1 - node path under /var/logs to download
+# $2 - local output path
+# $3 - node name
+# $4 - log file name
+paths=(kube-apiserver)
+for path in "${paths[@]}" ; do
+  output_dir="${BASE_COLLECTION_PATH}/startup_monitor_logs/$path"
```

Comment:
would rather generalize this to `staticpods/kube-apiserver/{startup,termination}.log`

Reply:
it is easier to copy to one file, at the moment it will be `must-gather/kube-apiserver/`

Reply:
we could change it to `must-gather/kube-apiserver/lifecycle_logs`